src/messages.json

```diff
@@ -208,6 +208,14 @@
         "Scene.outOfBounds": "Scene, in {world}, {numColumns} by {numRows} grid with {character} outside of the scene {relativeDirection} the scene, facing {direction}",
         "Sketchpad.character": "the robot",
         "Sketchpad.label": "A blank sketchbook with grid lines.  Your character in this scene is a Robot.",
+        "SoundOptionsModal.allSounds": "All Sounds",
+        "SoundOptionsModal.musicalSounds": "Musical Sounds",
```

Comment:
We're not entirely consistent about it, but we use alphabetical ordering for a lot of the message keys.  These are almost alphabetical except for this one. 

Alphabetical obviously makes sense to me, but I can also see the sense in putting them it in order by when they're used in the component.  Tthat would match what we do with CSS.

Either way I think it should be organised a little better.

Reply:
I think it makes sense to just alphabetize everything in messages.json, as the usage order doesn't matter in the file. I updated SoundOptionsModal to be alphabetical order for now.